docs/pages/docs/build-platforms/github-actions.mdx

```diff
@@ -64,7 +77,6 @@ If you are publishing to the Github Package Registry you will need to change the
     npx auto shipit
 ```
 
-This will use your `GITHUB_TOKEN` to publish to the Github Package Registry.
 The `NODE_AUTH_TOKEN` variable is required the `.npmrc` that `actions/setup-node@v1` sets up.
```

Comment:
Small typo noticed: a missing "for" i.e. "is required **for** the `.npmrc` that..."